tensorflow/core/grappler/devices.cc

```diff
@@ -67,7 +67,7 @@ int GetNumAvailableGPUs(
 }
 
```

Comment:
can we take this opportunity to fill out the #else block at line 59 with the ROCm equivalent implementation?

Reply:
Changed to report the number of GPUs, ignoring the "CUDA compute capability" argument.